semanticcpg/src/main/scala/io/shiftleft/semanticcpg/sarif/SarifConfig.scala

```diff
@@ -25,8 +25,7 @@ import java.net.URI
   * @param resultConverter
   *   A transformer class to map from Finding nodes to a SARIF `Result`.
   * @param customSerializers
-  *   Additional JSON serializers for any additional properties for [[io.shiftleft.semanticcpg.sarif.Sarif]] derived
```

Comment:
Linking to Sarif did not work at all. Its a sealed trait inside `io.shiftleft.semanticcpg.sarif.SarifSchema` but can't be addressed.
I tried:
`io.shiftleft.semanticcpg.sarif.SarifSchema.Sarif` and also `io.shiftleft.semanticcpg.sarif.SarifSchema$$Sarif`